models/blobs/ethereum/enriched.sql

```diff
@@ -0,0 +1,91 @@
+{{ config(
+    schema = 'blobs_ethereum',
+    alias = 'blobs_enriched',    
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash','blob_index'],
```

Comment:
this should match the unique check labels, this may be what's causing the incremental to fail:

tx_hash, blob_index, blob_kzg_commitment